chromium_src/ui/views/controls/button/md_text_button.cc

```diff
@@ -39,11 +133,76 @@ class BraveTextButtonHighlightPathGenerator
 
 namespace views {
 
-// To be called from MdTextButtonBase::UpdateColors().
-void MdTextButtonBase::UpdateColorsForBrave() {
+MdTextButton::MdTextButton(PressedCallback callback,
+                           const std::u16string& text,
+                           int button_context)
+    : MdTextButtonBase(std::move(callback), text, button_context) {
+  SetCornerRadius(100);
+  views::HighlightPathGenerator::Install(
+      this, std::make_unique<BraveTextButtonHighlightPathGenerator>());
+  auto* ink_drop = views::InkDrop::Get(this);
+  views::InkDrop::UseInkDropForFloodFillRipple(ink_drop,
+                                               /*highlight_on_hover=*/false,
+                                               /*highlight_on_focus=*/true);
+  ink_drop->SetCreateHighlightCallback(base::BindRepeating(
+      [](Button* host) {
```

Comment:
optional: if you bind corner_radius here, you could avoid static_cast<> below.

Reply:
That's probably nicer, but I might leave this as is (it hasn't actually changed from what was here before) just to have a nicer diff.

Reply:
Yeah sure. No problem.